docker/pg_tests/scripts/tests/daemon_test.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+set -e -x
```

Comment:
You need to make sure that this test is included in [the CI pipeline](https://github.com/wal-g/wal-g/blob/e7ffb57e4c5ba385ed596e45295ad8b60eb1ae76/.github/workflows/dockertests.yml#L103). It means that you also need to add the corresponding configuration files. Check how others Postgresql Docker tests are implemented for reference.